frontend/src/pages/auth/components/not-found/not-found.tsx

```diff
@@ -0,0 +1,26 @@
+import { useRouteError } from 'react-router-dom';
+
+import styless from './not-found.module.scss';
+
+interface RouteError {
+  statusText?: string;
+  message?: string;
+}
+
+const ErrorPage = (): JSX.Element => {
+  const error = useRouteError() as RouteError;
```

Comment:
As I understand error would have status and within status we may check if it's actual `404` - let's provide actual Not Found page on `404`, because currently this component looks like generic error page